### DISCUSSION TITLE
Phase Number / Phase split

### DISCUSSION THREAD

**Discussion by Dodge007** [tbnobody/OpenDTU#2158] [Ideas]
Hello i hope i write this understandable because im from austria and im not perfekt in english

ist it possible to tell the stu on what phase each inverter is, i hav 3HMS 1800 each one is on its one phase , i live in austria with 3phase elektrical,

it would be nice to see and set on what phase each inverter is and also to read out each phase of production, so that i can calculate the production and cosumption of my house correctly

so is it possible to implement this feature or this features
i hope im not the only one that could use this feature

Thanks

kevin

**Comment by stefan123t** [comment]
@Dodge007 Kevin the OpenDTU and other DTUs currently do not distinguish between the Phase your inverter is connected to. The OpenDTU will calculate the total Power generated across all Phases. You may either use an external Home Automation System (via MQTT or REST API) to collect the data from OpenDTU and calculate your per phase Totals.
Apart from that you may want to consider OpenDTU-OnBattery which has an option to read the values from a Power Meter and even use a Dynamic Power Limiter to reduce your export of energy to the legally allowed maximum of 600/800W.
See here for an example configuration: https://github.com/tbnobody/OpenDTU/issues/272#issuecomment-2219644352

**Reply by Dodge007** [reply]
Thanks for the fast response, i understand that and hope it will be implementet in the future , till then i will use a meter to see production of each phase,
im happy that i dont need the power limit funktion because my PV production i full registratet and produce legally up to 7Kwp into the grid.

would or is it possible to read the generatet power of each inverter? so that i can calculate each phase myself in NodeRed
so that im not in need of a meter?

Thanks

**Comment by super-mertl** [comment]
One step in this direction could be a selection for single phase inverters (eg HMS-2000).
In live view there is always shown "phase 1".
All my HMS are connected to phase 2 or 3 because my old inverter is connected to phase 1.
Another solution for this optical issua could be another text like overview / total production / ....
